string/TextCursor.test.ts

```diff
@@ -132,7 +132,7 @@ describe("inspect", () => {
     assertEquals(
       stripColor(new TextCursor(longStr, 37).inspect({ maxLength })),
       dedent(`
-        … jumps over the lazy dog then does it…
+        …·jumps·over·the·lazy·dog·then·does·it…
```

Comment:
Its fine as is, there is just a funny grammar thing...should it be "then it does" instead of "then does it"?

Reply:
Totally get the confusion. It's a clip from this string, which in full is not grammatically incorrect. It just got clipped in an unfortunate location.

> "the quick brown fox jumps over the lazy dog then does it again and again until it is tired"